index.html

```diff
@@ -38,13 +30,26 @@
                 <li>Contact</li>
                 <li><i class="bi bi-envelope emailsend"></i></li>
             </ul>
-            <i class="bi bi-list menu"></i>
+            <i id="Deploy_menu" class="bi bi-list menu"></i>
+            <ul id="mobile-menu">
+                <li id="close-btn" class="close-mobile-menu "><i class="bi bi-x-lg"></i></li>
+                <li class="line_border">
+                    <a href="#proyects">Portofolio</a>
+                </li>
+                <li class="line_border">
+                    <a href="#aboutme">About Me</a>
+                </li>
+                <li class="line_border">
+                    <a href="#contact">
+                        Contact Me
+                    </a>
+                </li>
+            </ul>
```

Comment:
- [ ] OPTIONAL
To improve the **user experience** it will be good practice to ensure that the _hamburger_ and the _close button_ are behaving as buttons when the mouse hovers on them. We can improve upon this aspect by adding the CSS attribute `cursor: pointer` to the _hamburger_ and the _close button_ on _mouse hover_ events. We can additionally change the color or opacity of these objects during the hover event.

Reply:
I add in the classes the cursor pointer and the opacity is 0.5 on both buttons